src/html.js

```diff
@@ -13,6 +13,14 @@ export default function HTML(props) {
           name="viewport"
           content="width=device-width, initial-scale=1, shrink-to-fit=no"
         />
+        <link
+          rel="stylesheet"
+          href="https://cdn.jsdelivr.net/npm/docsearch.js@2.6.2/dist/cdn/docsearch.min.css"
+        />
+        <script
+          type="text/javascript"
+          src="https://cdn.jsdelivr.net/npm/docsearch.js@2.6.2/dist/cdn/docsearch.min.js"
+        />
```

Comment:
Should we not import these files from our JS entry point to make them available in a single request?

Reply:
I did it like that at first, but I started getting build errors with `window is undefined`, so for now I moved it back there. I'd like to research it a little later, probably in separate PR.

But I think that I can return css without problem.

Reply:
I see! This is probably due to serverside rendering.

In that case, there's 2 options:

1. Use the client entrypoint (maybe import it from gatsby-browser.js if you can't find it)
2. Use a conditional require. `if (typeof window === 'object') require('...')`.

Reply:
Either way, sure, it's not a big deal.

Reply:
Yep, it's definitely SSR. I thought about using `gatsby-browser.js` to fix it, but decided to leave it as is for now, until I make other stuff work.

Reply:
Maybe in another PR. No big deal I think.

Reply:
please, create a ticket for this

Reply:
Done https://github.com/iterative/dvc.org/issues/1046